webhook_to_fedora_messaging/config/defaults.py

```diff
@@ -0,0 +1,40 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+class Defaults:
+    DEBUG = True
```

Comment:
I'm not sure it's safe to set `DEBUG` to `True` by default. If someone forgets to set it to `False` in production, it opens a remote execution vulnerability on the server from any error page. This seems a bit too dangerous.